.eslintrc.js

```diff
@@ -129,8 +129,9 @@ module.exports = {
         'plugin:@typescript-eslint/recommended-requiring-type-checking',
         'plugin:prettier/recommended',
       ],
-      plugins: ['@typescript-eslint', 'jsx-a11y', 'no-only-tests'],
+      plugins: ['@typescript-eslint', 'jsx-a11y', 'no-only-tests', 'blade'],
```

Comment:
This will change too right?
```suggestion
      plugins: ['@typescript-eslint', 'jsx-a11y', 'no-only-tests', 'eslint-plugin-blade'],
```

Reply:
@chaitanyadeorukhkar both works, ESLint will automatically add the `eslint-plugin` prefix.

Reply:
Okay